.github/workflows/docs.yaml

```diff
@@ -19,14 +19,15 @@ jobs:
 
       - name: Install dependencies
         run: |
+          pip install ghp-import
           sudo apt-get install --no-install-recommends -y graphviz make wget
```

Comment:
Doesn't it need a `sudo apt-get update` before that?

Reply:
Yes, you are right

Reply:
Updated